src/ResourceManager/AnalysisServices/Commands.AnalysisServices/Commands/NewAzureRmAnalysisServicesServer.cs

```diff
@@ -92,7 +97,13 @@ public override void ExecuteCmdlet()
                     }
                 }
 
-                var createdServer = AnalysisServicesClient.CreateOrUpdateServer(ResourceGroupName, Name, Location, Sku, Tag, Administrator);
+                var availableSkus = AnalysisServicesClient.ListSkusForNew();
```

Comment:
We have multiple rollouts that has slightly different skus available for customers.